api/V1/Recording.js

```diff
@@ -216,4 +217,80 @@ module.exports = (app, baseUrl) => {
         rows: result.rows,
       });
     });
+
+    /**
+    * @api {get} /api/v1/recordings/:id Get a recording
+    * @apiName GetRecording
+    * @apiGroup Recordings
+    * @apiDescription This call returns a recording in a JSON format and a JSON
+    * Web Token (JWT) which can be used to retrieve a specific audio recording.
```

Comment:
Not just audio right? How about "... can be used to retrieve the recorded content."

Reply:
Yes! I'll change that..